deployment/ecs.py

```diff
@@ -384,11 +415,22 @@ def get_task_definition(self, task_definition):
         return task_definition
 
     def update_task_definition(self, task_definition):
+        fargate_td = {}
+        if task_definition.requires_compatibilities and 'FARGATE' in task_definition.requires_compatibilities:
+            fargate_td = {
+                'execution_role_arn': task_definition.execution_role_arn or u'',
+                'requires_compatibilities': task_definition.requires_compatibilities or [],
+                'network_mode': task_definition.network_mode or u'',
+
+            }
         response = self._client.register_task_definition(
             family=task_definition.family,
             containers=task_definition.containers,
             volumes=task_definition.volumes,
-            role_arn=task_definition.role_arn
+            role_arn=task_definition.role_arn,
+            cpu=task_definition.cpu,
```

Comment:
CPU and memory in task definition should be for `fargate` only? Shouldn't this be same as `register_task_definition`?

Reply:
```
File "/c/Users/shoan/PycharmProjects/cloudlift/deployment/ecs.py", line 433, in update_task_definition
    **fargate_td
TypeError: register_task_definition() missing 2 required positional arguments: 'cpu' and 'memory'
```